ExerciseTwo/exercise.html

```diff
@@ -3,24 +3,21 @@
     Fix the 12 lines with indentation issues below.
 -->
 <!DOCTYPE html>
-<head>
-            <title>Some title</title>
+<html>
+  <head>
+      <title>Some title</title>
     </head>
```

Comment:
This needs to be vertically aligned with `<head>`.
This will then effect `<body>` below and everything underneath it.
@blam-codes